### PR TITLE
Add missing dnscache package

### PR DESCRIPTION
**Pull request by otofune**
This is my fault at #407.
And add descriptions about ENABLE_DNSCACHE feature.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -5323,7 +5332,8 @@
         },
         "ansi-regex": {
           "version": "2.1.1",
-          "bundled": true
+          "bundled": true,
+          "optional": true
```
Comment (otofune) [inline package-lock.json]: I can't realize why to change this..... 🤔 

My Node.js environment is described in the following:
```
[I] ~/.g/g/c/crowi > node --version                                       master
v10.16.0
[I] ~/.g/g/c/crowi > npm --version                                        master
6.9.0
```

Comment (hiroppy) [reply package-lock.json]: package-lock.json in master is old, so your change is correct.

**Comment by sotarok** [comment]
@otofune Sorry I changed the base branch to `wip-v1.7.7` and can you update package.json / package-lock.json again?

File: package-lock.json (hunk 1 above)
Comment (sotarok) [reply package-lock.json]: @otofune But Crowi `v1.7.x` supports Node.js v8. And Node.js v8.16.0 uses npm 6.4.0.
Can you confirm the version?
(We will move Node.js v10 with Crowi v1.8)

Comment (otofune) [reply package-lock.json]: !! 
I'll re-generate package.json with node v8 environment.

**Comment by otofune** [comment]
✅  I ran `npm install` with latest `node@8` docker container have following versions.

```
# node -v
v8.16.0
# npm -v
6.4.1
```

**Pull request merged**